tests/cypress/integration/forms_spec.js

```diff
@@ -0,0 +1,254 @@
+/// <reference types="cypress" />
+import {
+  standardFormUrls,
+  interactiveForms,
+  formsWithEmailTestId,
+} from "../utils";
+
+context("Marketo forms", () => {
+  beforeEach(() => {
+    cy.intercept({
+      method: "POST",
+      url: "/marketo/submit",
+    }).as("captureLead");
+  });
+
+  afterEach(() => {
+    cy.wait("@captureLead").then(({ request, response }) => {
+      expect(request.method).to.equal("POST");
+      expect(response.statusCode).to.equal(302);
+    });
+  });
+
+  // test static forms on /contact-us page
```

Comment:
What do you think about wrapping relevant tests in context instead of relying on a comment?

Also - beforeEach/afterEach hooks could be placed outside of context to avoid repetition and excessive nesting.

```
beforeEach(() => {
  cy.intercept({
    method: "POST",
    url: "/marketo/submit",
  }).as("captureLead");
});

afterEach(() => {
  cy.wait("@captureLead").then(({ request, response }) => {
    expect(request.method).to.equal("POST");
    expect(response.statusCode).to.equal(302);
  });
});

context("Interactive marketo forms", () => {
(...)
})

context("Static marketo forms", () => {
(...)
})


```

https://docs.cypress.io/guides/core-concepts/writing-and-organizing-tests#Test-Structure